cfgov/jinja2/v1/_includes/organisms/filterable-list-controls.html

```diff
@@ -210,13 +211,55 @@
                 {{ notification('warning', true,
                     'Sorry, there were no results based on your filter selections.',
                     'Please reset the filter or change your selections and try again.') }}
+            {% elif posts.paginator.count == 1 %}
```

Comment:
We pass [`django.template.defaultfilters.pluralize`](https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#pluralize) into our context [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/__init__.py#L58), so you could use something like `posts.paginator.count|pluralize:"s"`. Not sure that'd be nicer code or not.